src/clucie/utils.clj

```diff
@@ -1,8 +1,26 @@
 (ns clucie.utils)
 
+(defn kw-or-symbol? [x]
```

Comment:
May be better to name `keyword-or-symbol?`.

Reply:
@federkasten done 👍